pixeltable/catalog/catalog.py

```diff
@@ -506,7 +506,7 @@ def drop_dir(self, path: Path, if_not_exists: IfNotExistsParam, force: bool) ->
             raise_if_not_exists=if_not_exists == IfNotExistsParam.ERROR and not force,
         )
         if schema_obj is None:
-            _logger.info(f'Directory {path!r} does not exist, skipped drop_dir().')
+            _logger.info(f'Directory {str(path)!r} does not exist, skipped drop_dir().')
```

Comment:
comma splice (comma should be a semicolon)

Reply:
Done